admin/class-wp-security-bp-admin.php

```diff
@@ -59,15 +60,6 @@ class WP_Security_BP_Admin {
 	 */
 	private $hook_suffix;
 
```

Comment:
this should be in other line but it's to illustrate intellisense.

![intellisense-2](https://user-images.githubusercontent.com/43264089/53844481-233f9000-3fa6-11e9-87d9-23b304d4b7da.JPG)